### ISSUE TITLE
TypeError: Cannot read property 'normalize' of undefined

### ISSUE THREAD

**Issue by samselikoff** [shipshapecode/ember-prism#42]
Fresh install, hit this error which traced to this line

```js
code = Prism.plugins.NormalizeWhitespace.normalize(code);
```

I fixed it by adding this to my `ember-cli-build.js`:

```js
"ember-prism": {
  plugins: ["normalize-plugins"]
}
```

I guess this should be the default?

Thanks for the great addon!

**Comment by rwwagner90** [comment]
@samselikoff this is added in index.js, so not sure why it is not working https://github.com/shipshapecode/ember-prism/blob/master/index.js#L22

**Comment by rwwagner90** [comment]
Fixed in 0.6.1. I made sure the plugin exists before calling it. Unsure why the issue was popping up in the first place, but this should stop it.
